resources/schemas/org.gnome.shell.extensions.pano.gschema.xml

```diff
@@ -65,7 +65,7 @@
       </description>
     </key>
     <key name="exclusion-list" type="as">
-      <default>['Bitwarden', '1Password', 'KeePassXC']</default>
+      <default>['Bitwarden', '1Password', 'KeePassXC', 'org.gnome.World.Secrets']</default>
```

Comment:
Thes are `WM_CLASS` property of the window. You can check it with `xprop` command. It should be `secrets`.

![image](https://user-images.githubusercontent.com/1043714/189529102-6607dda1-e560-49b6-8587-060f0cc9a94d.png)

Reply:
When I set it to secrets it still captures what i copy
![image](https://user-images.githubusercontent.com/79922836/189529408-240ff967-588c-4061-99bd-0d158acebcb4.png)
But when it is set to org.gnome.World.Secrets it doesn't
![image](https://user-images.githubusercontent.com/79922836/189529501-7c50eb75-9e29-423f-ba71-dd1d62741ea3.png)

Reply:
Can you share your `xprop` output?

Reply:
I think it has something to do with wayland. I'm using it so I can't use xprop but if i use lg it says that wm_class is org.gnome.World.Secrets.
![image](https://user-images.githubusercontent.com/79922836/189529852-7ab52ffe-e6f9-4f35-b192-d14cd5dd6887.png)

Reply:
oh. That makes sense. I am on X11. I think we should add both `secrets` and `org.gnome.World.Secrets` then.